Makefile

```diff
@@ -158,3 +158,21 @@ verify: api-verify backend-verify frontend-verify
 .PHONY: yarn-ensure # Install the pinned version of yarn.
 yarn-ensure:
 	@./tools/install-yarn.sh
+
+.PHONY: k8s-start # Start a local k8s cluster
+k8s-start:
+	tools/kind.sh create cluster --name clutch || true
+	tools/kind.sh get kubeconfig --name clutch > $(PROJECT_ROOT_DIR)/build/kubeconfig-clutch
+	tools/kube-seed.sh
+
+	@echo
+	@echo "Run the follow command to export the environment variables:"
+	@echo '    eval $$(make k8s-env)'
```

Comment:
any need for this indirection? seems like we can just tell them to export the variable instead of telling them to eval this. clearer for user as well what's happening.

Reply:
in private we have several env vars to export, this was easier for that use case IMO. But for a single one we can just tell the user to do it them selves.